src/workloads/scale/BulkLoading.yml

```diff
@@ -0,0 +1,288 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
```

Comment:
Should we be the owners?

Reply:
Good question. I think we should be the owners since we implemented it. Therefore, we have the best knowledge of the rational behind the test.